src/api/payment-requests/legacy-payment-requests.md

```diff
@@ -149,6 +149,42 @@ digit voucher code that has $20 of loaded credit. The received code is only
+
+| Http code | Error code |                      Message                      |                                                                                                                        Description                                                                                                                        |
+| :-------- | :--------- | :------------------------------------------------ | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 404       | 3          | {% break _ TRANSACTION_NOT_FOUND %}               | The provided transaction doesn’t exist.                                                                                                                                                                                                                   |
+| 400       | 5          | {% break _ INVALID_REQUEST_ID %}                  | RequestId must be a valid UUID.                                                                                                                                                                                                                           |
+| 400       | 8          | {% break _ INVALID_AUTHORIZATION %}               | Authorization must be a non empty string.                                                                                                                                                                                                                 |
+| 400       | 17         | {% break _ REQUEST_CANCELLED %}                   | Action cannot be completed because the request has already been cancelled.                                                                                                                                                                                |
+| 400       | 18         | {% break _ REQUEST_EXPIRED %}                     | Action cannot be completed because the request has expired.                                                                                                                                                                                               |
+| 400       | 19         | {% break _ REQUEST_PAID %}                        | Action cannot be completed because the request has been paid.                                                                                                                                                                                             |
+| 403       | 21         | {% break _ FORBIDDEN %}                           | The Api Key provided doesn’t have the required permissions or the resource is not found.                                                                                                                                                                  |
```

Comment:
I added FORBIDDEN, BAD_REQUEST and INTERNAL_SERVER_ERROR as they were previously documented but it is perhaps unnecessary.